README.md

```diff
@@ -13,4 +13,4 @@ This task is similar to [Static List of TODOs](https://github.com/mate-academy/r
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_static-list-of-posts-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](zakhar1996.github.io/react_static-list-of-posts-js/) and add it to the PR description.
```

Comment:
Fix the demo link and update it in the PR description 
<img width="302" alt="image" src="https://github.com/user-attachments/assets/1ea47530-6340-4ad5-816d-d848a1a12141">
```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://zakhar1996.github.io/react_static-list-of-posts-js/) and add it to the PR description.
```